### PR TITLE
[MOB-900] Dark Mode support

### PR DESCRIPTION
**Pull request by jarrodrobins**
Uplifted the SDK to support Dark Mode. 

Additionally, several UI bugs have been fixed. Please see _PA SDK Dark Mode Implementation_ on Confluence for details.
Simulator Screen Shot - iPhone 13 - 2022-06-27 at 12 30 54
Simulator Screen Shot - iPhone 13 - 2022-06-27 at 12 30 57
Simulator Screen Shot - iPhone 13 - 2022-06-27 at 12 31 01
Simulator Screen Shot - iPhone 13 - 2022-06-27 at 12 31 11

### CODE REVIEW

File: Examples/Examples/Demo/WebViewController.h [hunk 1]
```diff
@@ -8,13 +8,9 @@
 
 #import <UIKit/UIKit.h>
 
-NS_ASSUME_NONNULL_BEGIN
```
Comment (JinW-Airwallex) [inline Examples/Examples/Demo/WebViewController.h:11]: 🤔 is this removal necessary?

Comment (jarrodrobins) [reply Examples/Examples/Demo/WebViewController.h:11]: Whoops, forgot to add `nonnull` to the properties. Removing the `NS_ASSUME_NONNULL_BEGIN` to be more explicit.

File: Examples/Examples/Cart/CartViewController.m [hunk 2]
```diff
@@ -94,14 +89,13 @@ - (void)setupSDK {
     [Airwallex setMode:mode];
 
     // Theme customization
-    UIColor *tintColor = [UIColor colorNamed:@"Purple Color"];
-    [AWXTheme sharedTheme].tintColor = tintColor;
+    //    UIColor *tintColor = [UIColor systemPinkColor];
```
Comment (JinW-Airwallex) [inline Examples/Examples/Cart/CartViewController.m:92]: should we remove these?

Comment (jarrodrobins) [reply Examples/Examples/Cart/CartViewController.m:92]: I thought about removing it, but they're using that `setupSDK` method as a sort of in-code documentation. It is mentioned in the actual documentation, so maybe it's not necessary?

**Review by JinW-Airwallex — APPROVED** [review]

File: Examples/Examples/Cart/CartViewController.m (hunk 2 above)
Comment (JinW-Airwallex) [reply Examples/Examples/Cart/CartViewController.m:92]: Yeah, ideally we should demonstrate how to customise the theme. Is it still possible with the changes?

**Comment by JinW-Airwallex** [comment]
Nice work Jarrod! ❤️
